Doc/library/pathlib.rst

```diff
@@ -1068,11 +1077,9 @@ call fails (for example because the path doesn't exist).
 
 .. method:: Path.is_dir(*, follow_symlinks=True)
 
-   Return ``True`` if the path points to a directory, ``False`` if it points
-   to another kind of file.
-
-   ``False`` is also returned if the path doesn't exist or is a broken symlink;
-   other errors (such as permission errors) are propagated.
+   Return ``True`` if the path points to an accessible directory, ``False``
+   otherwise. Use :meth:`Path.exists` to test if a path is accessible without
+   checking its type.
```

Comment:
```suggestion
   Return ``True`` if the path points to an accessible directory.
   Use :meth:`Path.exists` to determine if a path is not accessible,
   or the other ``is_*`` methods to test for other types.
```

How about this? Looking at it again, "False otherwise" seems redundant, and I think suggesting other `is_*` checks really emphasises that "`not is_dir()` doesn't imply `is_file()`".

Reply:
Added a different suggestion further down (or up... depending on which way the review reads...)